inscriptions/ISic001398.xml

```diff
@@ -199,14 +199,15 @@
             <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
                 <ab>
                     <lb n="1"/><persName type="attested"><name type="idionym">Ἑρμᾶ</name></persName> χρη
-                    <lb n="2" break="no"/><choice><corr>σ</corr><sic>ε</sic></choice>τὲ <space quantity="1" unit="character"/> χαῖρε
+                    <lb n="2" break="no"/>στὲ <space quantity="1" unit="character"/> χαῖρε
                     <lb n="3"/>ἔζησες <g ref="#ivy-leaf">❦</g>
                     <lb n="4"/>ἔτη <num value="28"><hi rend="supraline">κη</hi></num>
                 </ab>
             </div>
             <div type="apparatus">
                 <listApp>
-                    <app><note>text according to IG</note></app>
+                    <app><note>text according to IG. In line 2</note></app>
```

Comment:
Not sure what the significance of  'In line 2' is here - do you mean the rest of the text is not according to IG? Or should this simply be deleted?